theme/index.hbs

```diff
@@ -78,115 +83,337 @@
             body.classList.add('js');
         </script>
 
-        <input type="checkbox" id="sidebar-toggle-anchor" class="hidden">
+        <input type="checkbox" id="sidebar-toggle-anchor" class="hidden" checked>
+        <div id="page-wrapper" class="page-wrapper">
+            <div class="menu-bar">
+                <div class="menu-bar-inner menu-bar-left">
+                    <label id="sidebar-toggle" for="sidebar-toggle-anchor" class="menu-bar-button"
+                        title="Toggle Table of Contents" aria-label="Toggle Table of Contents" aria-controls="sidebar">
+                        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 -960 960 960" fill="currentColor">
```

Comment:
Don't just dump the whole SVG here

Reply:
I don't really like font awesome 4.0. Where else should I put the SVG? I prefer keeping it in line because it allows me to use the current color attribute. I don't have to statically set the color.

Reply:
Resolved because there is no workaround. I can't import any handlebar files. Also, I need the SVG inlined because it allows for the fill color to work.